src/api/app/controllers/webui/staging/excluded_requests_controller.rb

```diff
@@ -0,0 +1,60 @@
+module Webui
+  module Staging
+    class ExcludedRequestsController < WebuiController
+      layout 'webui2/webui'
+
+      before_action :require_login, except: [:index]
+      before_action :set_bootstrap_views
```

Comment:
```suggestion
      before_action :switch_to_webui2
```

Reply:
So this will turn up when grepping, once we are moving on to webui2 being default.